tests/lib/snaps/test-snapd-network-status-consumer/consumer.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Is this test meant to be python2?

If not please use `/usr/bin/python3` (or the env variant)